components/eamxx/tests/coupled/dynamics_physics/homme_mam4xx_pg2/CMakeLists.txt

```diff
@@ -72,25 +72,16 @@ foreach (file IN ITEMS ${TEST_INPUT_FILES})
   GetInputFile(${file})
 endforeach()
 
+# Compare output files produced by npX tests, to ensure they are bfb
+include (CompareNCFilesFamily)
 
-## Finally compare all MPI rank output files against the single rank output as a baseline, using CPRNC
-## Only if running with 2+ ranks configurations
-# This test requires CPRNC
-if (TEST_RANK_END GREATER TEST_RANK_START)
-  include (BuildCprnc)
-  BuildCprnc()
+CreateRange(MpiRanks ${TEST_RANK_START} ${TEST_RANK_END})
 
-  math (EXPR CMP_RANK_START ${TEST_RANK_START}+1)
-  foreach (MPI_RANKS RANGE ${CMP_RANK_START} ${TEST_RANK_END})
-
-    set (SRC_FILE "${BASE_TEST_NAME}_output.INSTANT.nsteps_x${NUM_STEPS}.np${MPI_RANKS}.${RUN_T0}.nc")
-    set (TGT_FILE "${BASE_TEST_NAME}_output.INSTANT.nsteps_x${NUM_STEPS}.np${TEST_RANK_START}.${RUN_T0}.nc")
-    set (TEST_NAME "${BASE_TEST_NAME}_np${TEST_RANK_START}_vs_np${MPI_RANKS}_output_bfb")
-    add_test (NAME ${TEST_NAME}
-              COMMAND cmake -P ${CMAKE_BINARY_DIR}/bin/CprncTest.cmake ${SRC_FILE} ${TGT_FILE}
-              WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
-    set_tests_properties(${TEST_NAME} PROPERTIES
-      LABELS "dynamics;physics;mam4xx"
-      FIXTURES_REQUIRED "${FIXTURES_BASE_NAME}_np${MPI_RANKS}_omp1;${FIXTURES_BASE_NAME}_np1_omp1")
-  endforeach()
-endif()
+CompareNCFilesFamily (
+  TEST_META_NAME ${TEST_BASE_NAME}_npMPIRANKS_vs_np1
+  FILE_META_NAME ${TEST_BASE_NAME}_output.INSTANT.nsteps_x${NUM_STEPS}.npMPIRANKS.${RUN_T0}.nc
+  MAGIC_STRING "MPIRANKS"
+  MAGIC_VALUES "${MpiRanks}"
+  LABELS dynamics physics mam4xx PEM
```

Comment:
I used the label "PEM" to follow the CIME tests names.

Reply:
What does "PEM" stand for?

Reply:
This is what CIME has to say about it:
```
PEM    modified pe counts mpi bfb test (seq tests)
       do an initial run with default pe layout                               (suffix: base)
       do another initial run with modified pes (NTASKS_XXX => NTASKS_XXX/2)  (suffix: modpes)
       compare base and single_thread
```
If you think the name is too cryptic, I can use something like MPI-BFB or something along that way? Any idea?

Reply:
I guess I would prefer less cryptic, but maybe PEM isn't cryptic to the majority of the project? The current test name/cmake function name with comment is probably good enough.